warehouse/static/sass/blocks/_package-header.scss

```diff
@@ -131,8 +135,18 @@
   }
 
   &__date {
-    font-style: italic;
-    margin-top: 20px;
-    padding-bottom: 5px;
+    color: $text-color;
+
+    span {
+      margin-top: 10px;
+
+      &:before {
+        border-color: rgba(0, 0, 0, 0.4);
```

Comment:
maybe best to use the [border color defined in colours.scss](https://github.com/pypa/warehouse/blob/master/warehouse/static/sass/settings/_colours.scss#L31)